src/main/resources/templates/pps/bankTransferSanctionsDetails.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+<head>
+    <title>
+        Pay by bank transfer - Confirmation statement
```

Comment:
I think we should change this title to be e.g. `Pay by bank transfer (Sanction: PN)`.
Normally the title would be the same as the large heading - but probably helpful to provide the above in the title as it can be found in the Matomo Analytics later on and it gets us away from saying variations of CS.

Referring to the link [Pay by bank transfer (Sanction: PN)](https://sanctions-payment-prototype-0d6d68d6c82d.herokuapp.com/v9/bank-transfer/sanctions-details)